Users/Docker_Image.rst

```diff
@@ -11,6 +11,16 @@ on your code in the working directory, like this:
 
     docker run --volume=$(pwd):/app --workdir=/app coala/base coala-ci
 
+In some computer operating systems and programming languages,
+an alias is an alternative and usually easier-to-understand or 
+more significant name for a defined data object.
+
+You can use only coala command with:
```

Comment:
This sentence is rather confusing. I think what you meant to say was "To run coala from docker using only the `coala` command, you can use:"